src/ar/gmanga/build.gradle

```
@@ -6,7 +6,7 @@ ext {
     extName = 'GMANGA'
     pkgNameSuffix = 'ar.gmanga'
     extClass = '.Gmanga'
-    extVersionCode = 11
+    extVersionCode = 14
```

only increment this by 1
```suggestion
    extVersionCode = 12
```

---

done 👍